diff --git a/src/util/00-mesa-defaults.conf b/src/util/00-mesa-defaults.conf
index b58b22f..1530fea 100644
--- a/src/util/00-mesa-defaults.conf
+++ b/src/util/00-mesa-defaults.conf
@@ -643,9 +643,7 @@ TODO: document the other workarounds.
             <option name="radeonsi_zerovram" value="true" />
         </application>
         <application name="Counter-Strike Global Offensive" executable="csgo_linux64">
-            <option name="mesa_glthread" value="true" />
             <option name="radeonsi_zerovram" value="true" />
-            <option name="radeonsi_clamp_div_by_zero" value="true" />
         </application>
         <application name="Rocket League" executable="RocketLeague">
             <option name="radeonsi_zerovram" value="true" />
