diff --git a/src/util/00-mesa-defaults.conf b/src/util/00-mesa-defaults.conf
index 7e48cff..7daaa6e 100644
--- a/src/util/00-mesa-defaults.conf
+++ b/src/util/00-mesa-defaults.conf
@@ -670,7 +670,6 @@ TODO: document the other workarounds.
         </application>
         <application name="Counter-Strike Global Offensive" executable="csgo_linux64">
             <option name="radeonsi_zerovram" value="true" />
-            <option name="radeonsi_clamp_div_by_zero" value="true" />
         </application>
         <application name="Rocket League" executable="RocketLeague">
             <option name="radeonsi_zerovram" value="true" />
